src/shared/logger/logger.ts

```diff
@@ -41,3 +43,11 @@ export function getLogger(): Logger {
 export function setLogger(logger: Logger | undefined) {
     toolkitLogger = logger
 }
+
+export function setMaxLogging() {
+    if (getLogger() instanceof WinstonToolkitLogger) {
```

Comment:
Consider throwing an error if getLogger is not expected instanceof. This will help future maintainers know something needs to happen in this space if they were to add a different logger impl.